src/main/java/com/avispl/dal/communicator/cisco/CiscoCommunicator.java

```diff
@@ -597,6 +597,123 @@ public void unmute() throws Exception {
+                    .collect(Collectors.toList());
+            long callsCount = connectedCalls.size();
+            if (callsCount > 1) {
+                throw new IllegalStateException(String.format("Ambiguous active calls found: %s, 1 expected. Unable to proceed.",
+                        callsCount));
+            }
+            if (callsCount == 0) {
+                if (logger.isDebugEnabled()) {
+                    logger.debug("Unable to populate media channels data: no active calls is available");
+                }
+                return;
+            }
+            Call activeCall = connectedCalls.get(0);
+            Channel[] callChannels = activeCall.getChannels();
+            if (callChannels == null || callChannels.length == 0) {
+                if (logger.isDebugEnabled()) {
+                    logger.debug("Unable to populate media channels data: no media channels data is available in the calls structure");
+                }
+                return;
+            }
+
+            AudioChannelStats audioChannelStats = new AudioChannelStats();
+            VideoChannelStats videoChannelStats = new VideoChannelStats();
+            CallStats callStats = new CallStats();
+            callStats.setCallId(activeCall.getCallId());
+            callStats.setRemoteAddress(activeCall.getRemoteNumber());
+            callStats.setProtocol(activeCall.getProtocol());
+
+            Arrays.stream(callChannels).forEach(channel -> {
+                String direction = channel.getDirection();
+                Audio[] audioChannels = channel.getAudio();
+                Optional<Audio> audioData = Arrays.stream(audioChannels).filter(a -> Objects.equals("Active", a.getStatus())).findFirst();
+
+                Video[] videoChannels = channel.getVideo();
+                Optional<Video> videoData = Arrays.stream(videoChannels).filter(a -> Objects.equals("Active", a.getStatus())).findFirst();
+
+                switch (direction){
+                    case "Incoming":
+                        if (audioData.isPresent()) {
+                            Audio audio = audioData.get();
+                            audioChannelStats.setJitterRx(audio.getJitter());
+                            audioChannelStats.setPacketLossRx(audio.getPacketLoss());
+                            audioChannelStats.setCodec(audio.getCodec());
+                        }
+                        if (videoData.isPresent()) {
+                            Video video = videoData.get();
+                            videoChannelStats.setJitterRx(video.getJitter());
+                            videoChannelStats.setPacketLossRx(video.getPacketLoss());
+                            videoChannelStats.setCodec(video.getCodec());
+                        }
+                    break;
+                    case "Outcoming":
```

Comment:
`Outcoming` is a strange word :)
Pointing out just in case it is not supposed to be `Outgoing`

Reply:
yeah there was a dumb mistake :)